src/data/sql-quiz.ts

```diff
@@ -831,26 +831,38 @@ const sqlQuiz = [
     Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
   },
   {
-    Question:
-      "What SQL statement contains a valid subquery?",
-    Answer: "SELECT * FROM employees WHERE wage > (SELECT AVG(wage) FROM employees)",
+    Question: "What SQL statement contains a valid subquery?",
+    Answer:
+      "SELECT * FROM employees WHERE wage > (SELECT AVG(wage) FROM employees)",
     Distractor1: "SELECT MAX(*) FROM employees",
-    Distractor2: "SELECT AVG(wage) FROM employees WHERE name IN ('name1', 'name2', 'name3')",
-    Distractor3: "SELECT country, AVG(wage) FROM employees GROUP BY country LIMIT 3",
+    Distractor2:
+      "SELECT AVG(wage) FROM employees WHERE name IN ('name1', 'name2', 'name3')",
+    Distractor3:
+      "SELECT country, AVG(wage) FROM employees GROUP BY country LIMIT 3",
     Explanation:
       "A SQL subquery is a query inside a query. So, in SQL, a subquery is also called a nested query or an inner query. The outer query in which the inner query is inserted is the main query.",
     Link: "https://www.freecodecamp.org/news/sql-subquery-how-to-sub-query-in-select-statement/"
   },
   {
-    Question:
-      "What is a good reason to use a temporary table?",
-    Answer: "to store and reuse intermediate query results to avoid redundant computations",
+    Question: "What is a good reason to use a temporary table?",
+    Answer:
+      "to store and reuse intermediate query results to avoid redundant computations",
     Distractor1: "to store data for a short-lived analysis such as 1-2 weeks",
     Distractor2: "to improve database security",
     Distractor3: "to improve query performance",
     Explanation:
       "A temporary SQL table is a table that is created and used within the context of a specific session or transaction in a database.",
     Link: "https://www.freecodecamp.org/news/sql-temp-table-how-to-create-a-temporary-sql-table/"
+  },
+  {
+    Question: "What does the bitwise AND operator do in SQL?",
+    Answer: "Performs AND operation on corresponding bits of two numbers.",
+    Distractor1: "Combines two strings.",
+    Distractor2: "Adds two numbers.",
+    Distractor3: "Multiplies two numbers.",
+    Explanation:
+      "In SQL, the operation 'BITWISE AND' is used to perform a bitwise AND operation between two values. This operation compares each bit of the two values and sets the corresponding bit in the result to 1 only if both bits are 1.",
+    Link: "https://www.freecodecamp.org/news/learn-sql-in-10-minutes/"
```

Comment:
this article goes more in depth

```suggestion
    Link: "https://www.freecodecamp.org/news/sql-operators-tutorial/#bitwise-operators"
```

Reply:
thank You for your suggesion but do i just have to Commit the suggesion that you have provided me.